### PR TITLE
Fix issue in `test_if_node_stop`

### PR DESCRIPTION
**Pull request by AureliaDolo**

### CODE REVIEW

File: massa-client/src/tests/scenarios.rs [hunk 1]
```diff
@@ -15,25 +21,40 @@ async fn run_client_cmd(cmd: &str) -> Result<String> {
...
-    std::thread::spawn(|| {
-        Command::new("cargo")
-            .arg("run")
-            .current_dir("../massa-node")
-            .unwrap()
-    });
+async fn test_run_node() {
+    let handle = run_node(Duration::from_secs(60 * 3));
+    sleep(Duration::from_secs(180)); // let it compile and start
+    let a = handle.join().unwrap();
```
Comment (yvan-sraka) [inline massa-client/src/tests/scenarios.rs:56]: But but ... I didn't get it ... if you join `massa-node` on the first `#[tokio test]` `#[serial]`, how do you run the full test suite on the same running node?

**Review by yvan-sraka — CHANGES_REQUESTED** [review]

File: massa-client/src/tests/scenarios.rs (hunk 1 above)
Comment (AureliaDolo) [reply massa-client/src/tests/scenarios.rs:56]: I just want that test to run once and then I will fill up that with the scenario

Comment (yvan-sraka) [reply massa-client/src/tests/scenarios.rs:56]: You want to write the whole client scenario inside the same test? Okay, I wasn’t thinking about that, and yes you're right that’s better :)

**Review by yvan-sraka — APPROVED** [review]

**Comment by yvan-sraka** [comment]
@AureliaDolo agreed to merge this one?

File: massa-client/src/tests/scenarios.rs (hunk 1 above)
Comment (AureliaDolo) [reply massa-client/src/tests/scenarios.rs:56]: For some commands we re use part of the output of the previous ones

**Comment by AureliaDolo** [comment]
> @AureliaDolo agreed to merge this one?

Not yet !

File: massa-consensus/src/settings.rs [hunk 2]
```diff
@@ -197,7 +201,7 @@ impl ConsensusSettings {
             initial_ledger_path: self.initial_ledger_path.clone(),
             block_reward: *BLOCK_REWARD,
             operation_batch_size: self.operation_batch_size,
-            initial_rolls_path: self.initial_ledger_path.clone(),
+            initial_rolls_path: self.initial_rolls_path.clone(),
```
Comment (AureliaDolo) [inline massa-consensus/src/settings.rs:204]: THIS IS IT !!!!!

**Review by yvan-sraka — APPROVED** [review]

**Comment by yvan-sraka** [comment]
bors merge

**Review by yvan-sraka — APPROVED** [review]

**Comment by yvan-sraka** [comment]
bors merge

**Pull request merged**